src/plugins/intel_cpu/CMakeLists.txt

```diff
@@ -150,3 +150,17 @@ endif()
 if(ENABLE_TESTS)
     add_subdirectory(tests)
 endif()
+
+if(ENABLE_CPU_EXTENSIONS)
+    if(NOT COMPILE_CPU_EXTENSIONS)
+        # use downloaded binary
+        find_package(cpu_extensions REQUIRED)
```

Comment:
for static libraries case we also need to add such find_package into OpenVINOConfig.cmake.in

Reply:
Updated.